tutorials/lab/coral2.rst

```diff
@@ -18,96 +18,82 @@ Things to Know
+    ]
+
 ------------------------
 Building Flux for CORAL2
 ------------------------
 
 The basic steps to building Flux for Cray Shasta systems are as follows:
 
-#.  :ref:`Build flux-core and flux-sched manually <manual_installation>`
-    with some prefix *P*.
+#.  :ref:`Build flux-core (version >= 0.49.0) and flux-sched manually
+    <manual_installation>` with some prefix *P*.
 #.  Build `flux-coral2 <https://github.com/flux-framework/flux-coral2>`_
     with the same prefix *P*.
-#.  Create a Flux config file specifying that the ``cray_pals_port_distributor.so``
-    plugin should be loaded with some given port range (see below for an example).
-    If you have other config files, put the new file in with the others.
-    Before launching Flux, point Flux to the *directory* containing your config
-    file(s) by setting the ``FLUX_CONF_DIR`` environment variable, or by passing
-    ``-o"-c/path/to/config"`` to ``flux start``.
-#.  As an alternative to creating a config file and setting ``FLUX_CONF_DIR``,
-    you can, after starting Flux, execute ``flux jobtap load
-    cray_pals_port_distributor.so port-min=$N port-max=$M`` for some *N* and *M*.
-
-
-If you see job failures with an error message like "no cray_pals_port_distribution
-event posted", check that you have the ``cray_pals_port_distributor.so`` plugin
-loaded by running ``flux jobtap list``. If you don't see it in the list, retry
-step 3 or 4 above.
-
-A script to build Flux is below.
-
-.. code-block:: sh
-
-  #!/bin/bash
-
-  set -e
-
-  PREFIX=$HOME/local  # a good default, but modify as needed
-  PORT_MIN=11000      # a good default, but modify as needed
-  PORT_MAX=12000      # a good default, but modify as needed
-
-  # Step 1: Build flux-core 0.29 or later
-
-  wget https://github.com/flux-framework/flux-core/releases/download/v0.29.0/flux-core-0.29.0.tar.gz
-
-  tar -xzvf flux-core-0.29.0.tar.gz && cd flux-core-0.29.0
-
-  ./configure --prefix=$PREFIX && make -j && make install && cd ..
-
-  # The `flux` executable will now be in ~/local/bin/flux but it needs some
-  # additional flux-coral2 extensions
-
-
-  # Step 2: Build flux-sched 0.18 or later (optional but recommended)
 
-  wget https://github.com/flux-framework/flux-sched/releases/download/v0.18.0/flux-sched-0.18.0.tar.gz
+------------------
+Flux with Cray PMI
+------------------
 
-  tar -xzvf flux-sched-0.18.0.tar.gz && cd flux-sched-0.18.0
+Applications linked to Cray MPICH will work natively with Flux
+provided the Cray MPICH library uses the PMI2 protocol instead of
+the homespun Cray PMI and libPALS. For Flux to support libPALS,
+flux-coral2 must be built (see above) and Flux must be configured
+to offer libPALS support. This is done by setting the "pmi" shell
+option to include "cray-pals" on a per-job basis like so:
 
-  ./configure --prefix=$PREFIX && make -j && make install && cd ..
+``flux submit -n1 -opmi=cray-pals hostname``
```

Comment:
Should we put this in code block so it better stands out?
```suggestion
.. code-block:: console

    $ flux submit -n1 -opmi=cray-pals hostname
```

And a tiny detail I just noticed - I think a lot of the other guides put the `$` to indicate the console - we should decide either way (and then make sure they are all consistent!)

Reply:
Good point, will fix.